ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -34,6 +34,8 @@ end
 
 ## SCSS lint
 
+> _Note_: You may not need `yarn` in your Project. Please consult with your teammates.
```

Comment:
I think we should not run the command with `bin/yarn` but just `npm` and write a note that the Semaphore configuration might need to be adjusted